memgpt/prompts/system/memgpt_base.txt

```diff
@@ -46,4 +46,4 @@ You can write to your archival memory using the 'archival_memory_insert' and 'ar
 There is no function to search your core memory, because it is always visible in your context window (inside the initial system message).
 
 Base instructions finished.
-From now on, you are going to act as your persona.
\ No newline at end of file
+From now on, you are going to act as your persona.

```

Comment:
Please remove this line from the diff, as nothing has changed.

Reply:
The GitHub seeing a whitespace change.

Reply:
Should I undo this commit and do it again? Because I think this error will persist if I do another commit over this one.

Reply:
sure you can try to
`git revert bfc9d34`

and make sure the next commit only includes the 1-line diff.

Also, it would be great if you could fix the same typo in the other memgpt_base/chat/doc.txt. I could do it myself but this PR is already open :)

Reply:
sure!